exe/etcd-env

```diff
@@ -40,10 +44,11 @@ def fetch(uri_str, limit = 10)
 end
 
 if __FILE__ == $PROGRAM_NAME
-  key, *commands = ARGV
+  key_str, *commands = ARGV.join(" ").split(" -- ")
+  keys = key_str.split(" ")
   if commands.empty?
     usage
     exit 1
   end
-  exec env_vars(key), *commands
+  exec env_vars(keys).reduce({}, :merge), *commands
```

Comment:
I think it would be cleaner to move the merge logic inside the `env_vars` function